src/logger.js

```diff
@@ -3,6 +3,10 @@ export default {
     console.log('DEBUG', ...args)
   },
 
+  debug(...args) {
```

Comment:
why are there now two functions that both are implemented as `console.log('DEBUG', ...args)`? I guess then the log function should have another string than 'DEBUG' printed

Reply:
I agree, I'll remove the debug and we leave it like it was